Makefile

```diff
@@ -36,16 +37,23 @@ html:
 pdf:
 	make -C doc latexpdf
 
+PYENV_DIR=.virtualenv
 FORKED_FORMULAS_DIR=formulas
-FORMULAS=`python3 -c 'import sys; sys.path.append("scripts");from update_mrconfig import *; print(*get_org_repos(make_github_agent(), "salt-formulas"), sep="\n")'| egrep 'salt-formula-' | sed 's/salt-formula-//'`
+FORMULAS=`. $(PYENV_DIR)/bin/activate && python3 -c 'import sys; sys.path.append("scripts");from update_mrconfig import *; print(*get_org_repos(make_github_agent(), "salt-formulas"), sep="\n")'| egrep 'salt-formula-' | sed 's/salt-formula-//'`
+
+set_push:
+	(for formula in $${FORMULAS_DIR:-$(FORKED_FORMULAS_DIR)}/*; do FORMULA=`basename $$formula` && cd $$formula && git remote set-url --push origin git@github.com:salt-formulas/salt-formula-$$FORMULA.git && cd ../..; done)
 
 scripts_prerequisites:
-	pip3 install -r scripts/requirements.txt
+	@if ! which virtualenv; then echo "Please install virtualenv first";  exit 2; fi
+	@virtualenv -p python3 --no-site-packages $(PYENV_DIR)
+	@. $(PYENV_DIR)/bin/activate
+	$(PYENV_DIR)/bin/python -m pip install -r scripts/requirements.txt
 
 list: scripts_prerequisites
 	@echo $(FORMULAS)
 
-update_forks:
+update_forks: scripts_prerequisites
```

Comment:
Why update_forks is expected to install scripts requirements?

Reply:
its using virtualenv now, so it need's to run it each time just for sure..